backend/vpnathome/apps/management/models.py

```diff
@@ -3,8 +3,13 @@
 
 class Settings(models.Model):
 
+    FILE_EXTENSION_CHOICES = [('ovpn', 'OVPN'), ('conf', 'CONF')]
+
     email_enabled = models.BooleanField(default=False)
     registration_enabled = models.BooleanField(default=True)
+    config_file_extension = models.CharField(choices=FILE_EXTENSION_CHOICES,
```

Comment:
I think we'd need 2 options: one for the client (which mostly expects ovpn), one for server (which mostly requires `conf` file, as this is what `systemd` on Linux expects).

I'd then default to `ovpn` on the client and `conf` for the server.